File: fix-syntax.py
import re

with open('app/page.tsx', 'r', encoding='utf-8') as f:
    code = f.read()

# Bruteforce fix the div!
code = re.sub(
    r'<div className=\{max-w-\[1400px\] mx-auto w-full flex-1 flex flex-col  \}>',
    '<div className={max-w-[1400px] mx-auto w-full flex-1 flex flex-col  }>',
    code
)

with open('app/page.tsx', 'w', encoding='utf-8') as f:
    f.write(code)

File: fix-syntax.js
const fs = require('fs');
let code = fs.readFileSync('app/page.tsx', 'utf8');

code = code.replace(
  '<div className={max-w-6xl mx-auto w-full flex-1 flex flex-col }>',
  '<div className={max-w-6xl mx-auto w-full flex-1 flex flex-col }>'
);

fs.writeFileSync('app/page.tsx', code);
